workspaces/arborist/lib/arborist/load-actual.js

```diff
@@ -333,7 +333,7 @@ module.exports = cls => class ActualLoader extends cls {
 
   async #loadFSTree (node) {
     const did = this.#actualTreeLoaded
-    if (!did.has(node.target.realpath)) {
+    if (node.target && !did.has(node.target.realpath)) {
```

Comment:
Leaving this fix as it was in the initial commit for now, but I think `!node.isLink` is a more semantic check that will accomplish the same thing.